app/src/hooks/useToggle/useToggle.tsx

```diff
@@ -0,0 +1,10 @@
+import { useState } from "react";
+
+export const useToggle = (init: boolean) => {
+  const [isOpen, setIsOpen] = useState<boolean>(init);
+  const toggle = () => setIsOpen(!isOpen);
+  const open = () => setIsOpen(true);
+  const close = () => setIsOpen(false);
+
+  return { isOpen, toggle, open, close };
+};
```

Comment:
`open` and `close` are redundant.
Let's leave toggle only: 

```suggestion
export const useToggle = () => {
  const [isOpen, toggle] = useState(false);

  return { isOpen, toggle };
};
```